### PR TITLE
mobile: invite farcaster frame

### PR DESCRIPTION
**Pull request by dcposch**

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/InviteScreen.tsx [hunk 1]
```diff
@@ -267,6 +279,14 @@ function ReferralButtonsFooter({
     } else return "";
   })();
 
+  const shareFarcaster = () => {
+    console.log(`[INVITE] share on farcaster`);
+    const frameUrl = `https://daimo.com/frame/invite/${account.address}`;
+    const message = encodeURIComponent(frameUrl);
+    Linking.openURL(`https://warpcast.com/~/compose?text=${message}`);
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/screen/InviteScreen.tsx:286]: do you need `embeds[]=` here? https://docs.farcaster.xyz/reference/warpcast/cast-composer-intents

File: apps/daimo-web/src/app/frame/frameLinkService.ts [hunk 2]
```diff
@@ -93,7 +109,10 @@ export class FrameLinkService {
     viewerContext: UserViewerContext
   ): Promise<[boolean, string]> {
     const { auth, owner } = frame;
-    if (auth.fidMustBeBelow && user.fid > auth.fidMustBeBelow) {
+    if (auth.mustBePowerUser) {
+      console.log(`TODO power user check: ` + JSON.stringify(user));
```
Comment (nalinbhardwaj) [inline apps/daimo-web/src/app/frame/frameLinkService.ts:113]: ⚡ https://api.warpcast.com/v2/power-badge-users

File: apps/daimo-mobile/src/view/screen/InviteScreen.tsx [hunk 3]
```diff
@@ -47,8 +50,12 @@ export function InviteScreen() {
 function InviteScreenInner({ account }: { account: Account }) {
   const inviteLinkStatus = account.inviteLinkStatus;
 
+  console.log(
+    `[INVITE] render ${account.name}, ${JSON.stringify(inviteLinkStatus)}`
+  );
+
   const header =
-    account.invitees.length > 0 ? (
+    account.invitees.length > 0 || inviteLinkStatus?.isValid ? (
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/screen/InviteScreen.tsx:58]: intentional?

File: apps/daimo-web/src/app/frame/frameLinkService.ts (hunk 2 above)
Comment (dcposch) [reply apps/daimo-web/src/app/frame/frameLinkService.ts]: turns out user object has a power_badge property

**Pull request merged**